worlds/kid.wbt

```diff
@@ -89,6 +89,9 @@ DEF BLUE_1 NUgus {
   rotation 0 0 1 3.1415
   controller "nugus_controller"
   controllerArgs [
-    "32"
+    # TCP port to listen on
```

Comment:
Why is this in this PR?

Reply:
Was needed to test, and it doesn't seem to be in main.

Reply:
I think it would be easier to add here and merge. But we could move it to a separate PR.